site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -698,8 +696,15 @@ ${code}
 === Replacement Variables
 
 [.api]
-[field]#application# [type]#[Application]# [since]#Available since 1.28.0#::
+[field]#application# [type]#[Application]# [since]#Available since 1.46.0#::
```

Comment:
I had to go back to the old `fusionauth-api` repo. I don't think this object has ever been populated before now.